src/main/java/com/dev/cinema/config/MyWebAppInitializer.java

```diff
@@ -5,7 +5,8 @@
 public class MyWebAppInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
     @Override
     protected Class<?>[] getRootConfigClasses() {
-        return new Class[]{com.dev.cinema.config.AppConfig.class};
+        return new Class[]{com.dev.cinema.config.AppConfig.class,
+                com.dev.cinema.config.SecurityConfig.class};
```

Comment:
```suggestion
        return new Class[]{AppConfig.class, SecurityConfig.class};
```
can we make imports?

Reply:
fixed